### PR TITLE
Add OWASP Maryam to GSoC 2021 ideas

### PR DESCRIPTION
**Pull request by saeeddhqan**

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 1]
```diff
@@ -96,3 +96,33 @@ list? Great, please submit it!
 
 ----
 
+
+### [OWASP Maryam](https://github.com/saeeddhqan/maryam)
+
+##### Explanation of Ideas
+OWASP Maryam is a modular open-source OSINT based framework. Maryam is written in Python and It’s designed to provide a powerful environment to harvest data from open-sources and search-engines and collect data quickly and thoroughly.
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:103]:
```suggestion
OWASP Maryam is a modular open-source OSINT based framework. Maryam is written in Python and it’s designed to provide a powerful environment to harvest data from open-sources and search-engines and collect data quickly and thoroughly.
```

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 2]
```diff
@@ -96,3 +96,33 @@ list? Great, please submit it!
...
+
+##### Getting started
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:105]:
```suggestion
##### Getting Started
```

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 3]
```diff
@@ -96,3 +96,33 @@ list? Great, please submit it!
...
+
+* You may read the available documents in the [wiki page](https://github.com/saeeddhqan/maryam/wiki). Especially, Development Guide section.
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:107]:
```suggestion
* You may read the available documents in the [wiki page](https://github.com/saeeddhqan/maryam/wiki). Especially the Development Guide section.
```

File: pages/initiatives/gsoc/gsoc2021ideas.md [hunk 4]
```diff
@@ -96,3 +96,33 @@ list? Great, please submit it!
...
+* Any question, problem, and discussion? contact with [Saeed Dehqan](mailto:saeed.dehghan@owasp.org).
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2021ideas.md:108]: Is your email address actually a different spelling vs your name? (Below too)

I understand it totally could be, it just seemed out of place and caught my eye.

**Review by kingthorin — CHANGES_REQUESTED** [review]

**Comment by saeeddhqan** [comment]
Thank you

**Review by kingthorin — APPROVED** [review]

**Pull request merged**